### PR TITLE
Add Cloud SQL SQL Server connection pooling sample

### PR DESCRIPTION
**Pull request by jsimonweb**

### CODE REVIEW

File: cloud-sql/sql-server/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                        {
+                            Candidate = reader.GetString(0),
+                            TimeCast = reader.GetDateTime(1)
+                        });
+                    }
+                }
+            }
+            using (var countsCommand = _connection.CreateCommand())
+            {
+                countsCommand.CommandText = @"
```
Comment (SurferJeffAtGoogle) [inline cloud-sql/sql-server/Controllers/HomeController.cs]: Add a comment for this block of code.

File: cloud-sql/sql-server/Controllers/HomeController.cs [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+            if (validInput)
+            {
+                // [START cloud_sql_server_dotnet_ado_create]
+                insertTimestamp = DateTime.Now;
+                try
+                {
+                    using (var insertVoteCommand = _connection.CreateCommand())
+                    {
+                        insertVoteCommand.CommandText =
```
Comment (SurferJeffAtGoogle) [inline cloud-sql/sql-server/Controllers/HomeController.cs]: Add a comment.

File: cloud-sql/sql-server/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+        DbConnection InitializeDatabase()
+        {
+            DbConnection connection;
+            string database = Configuration["CloudSQL:Database"];
+            connection = GetSqlServerConnection();
+            connection.Open();
+            using (var createTableCommand = connection.CreateCommand())
+            {
+                createTableCommand.CommandText = @"
```
Comment (SurferJeffAtGoogle) [inline cloud-sql/sql-server/Startup.cs]: Add a comment.

File: cloud-sql/sql-server/Controllers/HomeController.cs (hunk 1 above)
Comment (jsimonweb) [reply cloud-sql/sql-server/Controllers/HomeController.cs]: Done.

File: cloud-sql/sql-server/Controllers/HomeController.cs (hunk 2 above)
Comment (jsimonweb) [reply cloud-sql/sql-server/Controllers/HomeController.cs]: Done.

File: cloud-sql/sql-server/Startup.cs (hunk 3 above)
Comment (jsimonweb) [reply cloud-sql/sql-server/Startup.cs]: Done.

**Pull request merged**